packages/twenty-server/src/engine/metadata-modules/userRole/userRole.service.ts

```diff
@@ -82,11 +122,72 @@ export class UserRoleService {
     });
   }
 
-  public async getRoleForUserWorkspace(
+  public async unassignRolesFromWorkspaceMember({
```

Comment:
```suggestion
  public async unassignAllRolesFromWorkspaceMember({
```
Probably more suited, at least for now? Except if we want to provide roleIds as parameters